src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -51,8 +52,9 @@ public final class FuzzTarget {
   private static Meta meta;
   private static String methodReference;
   private static Executable[] targetExecutables;
-  private static Map<Executable, Class<?>[]> throwsDeclarations;
-  private static Set<SimpleGlobMatcher> ignoredExceptionMatchers;
+  private static Object targetInstance;
+  private static Map<Executable, Class<?>[]> throwsDeclarations = Collections.emptyMap();
```

Comment:
Do we need the initializer here? We would probably want these to be set explicitly in any case and could fail instead if they are `null`.

Reply:
They are not needed in the JUnit case, at least not now, and initializing them in the overloaded `fuzzerInitialize` function seemed unnecessary complex. Nevertheless, I moved the initialization there to see how it would look like. WDYT?

Reply:
Looks good!

Reply:
Is the `throwsDeclarations` initializer still needed?